test/fixtures/wpt/resources/testdriver.js

```diff
@@ -290,7 +290,7 @@
         /**
          * Send a sequence of pointer actions
          *
-         * @returns {Promise} fufilled after actions are sent, rejected if any actions
+         * @returns {Promise} fulfilled after actions are sent, rejected if any actions
```

Comment:
Should this be fixed upstream? Or is it ok to fix it in our copy?

Reply:
I think it should be fixed upstream, AFAIK we use automated tooling to update those tests.